modules/ding_provider/tests/ding_provider_implementation.test

```diff
@@ -569,6 +569,7 @@ abstract class DingSearchProviderImplementationTestCase extends DingProviderImpl
     $query = ding_provider_invoke('search', 'prepare_reference_query', $test_query_3);
     $this->assertFalse($query->isFuzzy());
     $this->assertEqual($query->getMaterialFilter(), [$test_query_3]);
+    $this->assertTrue($query->isMaterialFilterInclude());
```

Comment:
Would it be possible to extend the test with a case where the material filter is exclude?